### DISCUSSION TITLE
Gridbuilder not adding display grid to grid container element

### DISCUSSION THREAD

**Discussion by khalilKhassep** [awcodes/filament-tiptap-editor#389] [Q&A]
While using grid builder i fount that its not adding display grid to its container even after isolating theme styles its still not adding it. 
I investegate Js files to know to to solve it i found that the Gridbuilder file dose not applay display grid there is a screen shoot for it. anyhelp for how to make it dynmic by not applying the display proparty from outer styles. 
![image](https://github.com/awcodes/filament-tiptap-editor/assets/47596595/9bea5f08-7660-4bbb-8559-7f21fdf9ef6e)

**Comment by awcodes** [comment]
This looks like custom code and not part of the plugin. What are you trying to accomplish?

**Reply by khalilKhassep** [reply]
Sorry for my weknes typing and description please apolgise me and try to understand me i will be thank full for you ill try descriping with screenshoots the bevhior 
![image](https://github.com/awcodes/filament-tiptap-editor/assets/47596595/05100309-4b58-4f8d-b961-fc95c0503294)
When inserting the grid and geting the content to front end its not adding display grid to filament-tiptap-grid-builder so its not applying the grid system because there is the default display for the element which is block so i have to add the proparty manualy by typing it in my custom styles and the code above i toke it from plugin js file named GridBuilder.js
my question is how to not adding the display grid from my custom style insted let the editot add it

**Comment by awcodes** [comment]
This plugin doesn't make any assumptions about how the rendered HTML should look, that's why it doesn't include the grid class. Some people might want it to be 'flex' instead of 'grid'.
